package.json

```diff
@@ -39,7 +39,6 @@
   "dependencies": {
     "array-move": "^1.0.0",
     "array-range": "^1.0.1",
-    "babel-runtime": "^6.26.0",
```

Comment:
turns out this wasn't used anymore since I removed the babel-transform-runtime plugin some time ago.